renderers/pregame.py

```diff
@@ -1,38 +1,30 @@
 from data.status import Status
 from rgbmatrix import graphics
 from utils import get_font, center_text_position
+from renderers.teams import TeamsRenderer
 from renderers.scrollingtext import ScrollingText
 import data.layout
 
 class Pregame:
-  def __init__(self, canvas, game, data, probable_starter_pos):
+  def __init__(self, canvas, game, scoreboard, data, probable_starter_pos):

```

Comment:
The places where this function is called need to be updated to pass in the scoreboard as a parameter